float32 SteeringWheel
float32 Ax
int32 GearNo
float32 VC_SwitchOn
int32 Lights_Hazard
int32 Lights_Indicator
